.github/workflows/release.yaml

```diff
@@ -81,8 +89,36 @@ jobs:
           GITHUB_TOKEN=${{ secrets.GIT_UPLOAD_TOKEN }} hub release edit -m "" -a "${tarball}" "${tag}"
           popd
 
+      - name: download-artifacts-arm64
```

Comment:
shouldn't this whole thing just be a for loop:
```
for arch in arm64 amd64 s390x; do
   ...
done
```

Reply:
To do this using GHA is not as simple as it'd be to do it in bash, unfortunately.